pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.pytest.ini_options]
+minversion = "6.0"
+filterwarnings = "ignore:html5lib:DeprecationWarning"
```

Comment:
As a nit, I'd put this in `setup.cfg` because we already have that. Either that or move the things that are in `setup.cfg` into here.

Reply:
Agreed. flake8 support is blocked on https://gitlab.com/pycqa/flake8/-/issues/428 so I'll move everything into `setup.cfg`

Reply:
Updated ce9bad8